pkgs/applications/video/aegisub/default.nix

```
@@ -55,63 +65,80 @@ let
 in
 stdenv.mkDerivation rec {
   pname = "aegisub";
-  version = "3.3.2";
+  version = "3.3.2-20220612";
 
   src = fetchFromGitHub {
     owner = "wangqr";
     repo = pname;
-    rev = "v${version}";
-    sha256 = "sha256-Er0g8fJyx7zjNVpKw7zUHE40hU10BdYlZohlqJq2LE0=";
+    rev = "91f8b5f91eb960bad19899c10af08aca34f9b697";
+    sha256 = "sha256-lPkPWSsncsBKJHDnma9cUXsQJynruT9JpPkMTHdQ/e8=";
   };
 
-  patches = [ ./no-git.patch ];
-
   nativeBuildInputs = [
-    autoreconfHook
     intltool
     luajit52
     pkg-config
     which
+    cmake
   ];
+
   buildInputs = [
     boost
+    expat
     ffmpeg
     ffms
     fftw
     fontconfig
     freetype
+    fribidi
+    glib
+    harfbuzz
     icu
     libGL
     libGLU
     libX11
     libass
     libiconv
     libuchardet
+    pcre
     wxGTK
     zlib
   ]
+  ++ lib.optionals stdenv.isDarwin [
+    CoreText
+    CoreFoundation
+    AppKit
+    Carbon
+    IOKit
+    Cocoa
+  ]
   ++ optional alsaSupport alsa-lib
-  ++ optional automationSupport luajit52
   ++ optional openalSupport openal
   ++ optional portaudioSupport portaudio
   ++ optional pulseaudioSupport libpulseaudio
   ++ optional spellcheckSupport hunspell
   ;
 
-  configureFlags = [
-    "--with-boost-libdir=${boost.out}/lib"
-    "--with-system-luajit"
-    "FORCE_GIT_VERSION=${version}"
-  ];
-
   enableParallelBuilding = true;
 
   hardeningDisable = [
     "bindnow"
     "relro"
   ];
 
-  postInstall = "ln -s $out/bin/aegisub-* $out/bin/aegisub";
+  patches = lib.optionals (!useBundledLuaJIT) [
+    ./remove-bundled-luajit.patch
+  ];
+
+  NIX_CFLAGS_COMPILE = "-I${luajit52}/include";
+  NIX_CFLAGS_LINK = "-L${luajit52}/lib";
+
+  configurePhase = ''
+    export FORCE_GIT_VERSION=${version}
+    mkdir build-dir
```

That being said, it is better to document it.

```suggestion
    # Workaround for a Nixpkgs bug; remove when the fix arrives
    mkdir build-dir
```

---

Documented.